ngraph/python/TUTORIAL.md

```diff
@@ -0,0 +1,216 @@
+Add_2 node has parent x
+Add_2 node has parent b
+```
+
+By analogy, `Node`'s children can be retrieved using the `outputs()` method which returns a list of `Output` objects. By contrast to inputs, each output can be connected to multiple downstream nodes.
+
+```python
+>>> for node_output in node.outputs():
+>>>     children = [target.get_node() for target in node_output.get_target_inputs()]
+>>>     print('{} node has children {}'.format(node.get_friendly_name(), ','.join([c.get_friendly_name() for c in children])))
+>>> 
+Add_2 node has children y
+```
+
+##### Shapes
+
+Each `Input` and `Output` has an associated tensor shape, which describes what data it can accept or produce. Shapes can be dynamic, which means that they are only partially known. You can get information about this shape by calling the `get_partial_shape` method. If the returned `PartialShape` object's `is_dynamic` property returns `True`, then the shape is known partially. Otherwise you can convert the shape to a fully known static shape (list of integers).
+
+```python
+partial_shape = node_input.get_partial_shape()
+if partial_shape.is_dynamic:
+    print('{} node has a dynamic shape {}'.format(node.get_friendly_name(), partial_shape))
+else:
+    shape = list(partial_shape.to_shape())
+    print('{} node has a static shape {}'.format(node.get_friendly_name(), shape))
+```
+
+##### Node attributes
+
+Nodes may also have additional attributes. For example a `Softmax` node has an `axis` attribute. Each attribute may be retrieved using a dedicated method, for example `node.get_axis()`.
+
+```python
+>>> node = ng.softmax(data=[[1, 2], [3, 4]], axis=1)
+>>> node.get_axis()
+1
+```
+
+Currently, you can get all attributes of a node using the `_get_attributes` method. Please note that this is an internal API method and may change in future versions of OpenVINO.
+
+The following code displays all attributes for all nodes in a function:
+
+```python
+for node in function.get_ordered_ops():
+    attributes = node._get_attributes()
+    if(attributes):
+        print('Operation {} of type {} has attributes:'.format(node.get_friendly_name(), node.get_type_name()))
+        for attr, value in attributes.items():
+            print("  * {}: {}".format(attr, value))
+    else:
+        print('Operation {} of type {} has no attributes.'.format(node.get_friendly_name(), node.get_type_name()))
+```
+
+##### Node runtime information
+
+Attributes are properties of nodes in the computational graph, which will be stored when the model is serialized to a file. However, there can be additional properties of nodes, which are only important during the execution of a graph. An example of such a property is `affinity`, which determines which operation will be executed on which hardware in a heterogeneous environment. You can get and set runtime information by using the `get_rt_info` method of a node.
```

Comment:
If we want to keep this section we need to say that rt info attribute are currently used only for internal needs and we do not guaranty backward compatibility for this attributes.

Reply:
Done.